subsys/bluetooth/host/gatt.c

```diff
@@ -674,14 +790,14 @@ static void sc_process(struct k_work *work)
 	sc->params.data = &sc_range[0];
 	sc->params.len = sizeof(sc_range);
 
-	if (bt_gatt_indicate(NULL, &sc->params)) {
+	if (bt_gatt_indicate(sc->conn, &sc->params)) {
```

Comment:
gatt_sc is a singleton. Should we maybe remove the CONTAINER_OF in this function, and use gatt_sc ?

Reply:
Done in latest push!